modelerfour/src/modeler/modelerfour.ts

```diff
@@ -1002,9 +1004,8 @@ export class ModelerFour {
           // but we're going to warn them anyway.
 
           this.session.warning(
-            `The schema '${
-              schema?.["x-ms-metadata"]?.name || name
-            }' with an undefined type and declared properties is a bit ambigious. This has been auto-corrected to 'type:object'`,
```

Comment:
Somehow Prettier doesn't pick up the right line length setting on my machine...

Reply:
how are you running it?

Reply:
I've got an Emacs package that launches Prettier against any TypeScript buffer on save, I need to look into how its loading up the Prettier config file